app/assets/src/components/views/SampleUploadFlow/UploadMetadataStep.jsx

```diff
@@ -102,13 +105,22 @@ class UploadMetadataStep extends React.Component {
           <div className={cs.controls}>
             <PrimaryButton
               text="Continue"
-              onClick={this.handleContinue}
+              onClick={withAnalytics(
```

Comment:
Probably also good to put a log step at line 57, which is where manual metadata gets validated and any issues are displayed.

Reply:
Good point. I'll move it inside handleContinue.